src/ExceptionHandling.java

```diff
@@ -32,12 +32,44 @@ public void getLength(String s){
             System.out.println("Exception cause : " + np.getMessage());
         }
     }
+
+    public int getArrayLength(int[] a){
+        try{
+            return a.length;
+        } catch (NullPointerException np){
+            System.out.println("Array is null : " + np.getMessage());
+        }
+        return 0;
+    }
+
+    // Making aware that this API could throw an exception. Possibly to be handled with try catch
+    public void getArrayIndexThrowException(int[] a, int index) throws ArrayIndexOutOfBoundsException {
+        System.out.println(a[index]);
+    }
+
+    //Handling the exception through try catch block
+    public void getArrayIndexHandleException(int[] a, int index) {
+        try{
+            System.out.println(a[index]);
+        } catch (ArrayIndexOutOfBoundsException e){
+            System.out.println(e.getMessage());
+        }
+    }
 }
 
 public class ExceptionHandling {
     public static void main(String[] args) {
         PerformOperations obj = new PerformOperations();
```

Comment:
I'd suggest to use meaningful names for object creation.

Reply:
Yes.